src/global/pid_lut/pid_lut.cc

```diff
@@ -1,62 +1,23 @@
 // SPDX-License-Identifier: LGPL-3.0-or-later
 // Copyright (C) 2022-2025 Christopher Dilks, Simon Gardner
 
-#include <Evaluator/DD4hepUnits.h>
 #include <JANA/JApplicationFwd.h>
 #include <JANA/Utils/JTypeInfo.h>
 #include <cmath>
 #include <string>
 #include <vector>
 
 #include "algorithms/pid_lut/PIDLookupConfig.h"
-#include "algorithms/pid_lut/PhaseSpacePIDConfig.h"
 #include "extensions/jana/JOmniFactoryGeneratorT.h"
 // factories
 #include "factories/pid_lut/PIDLookup_factory.h"
-#include "factories/pid_lut/PhaseSpacePID_factory.h"
 
 extern "C" {
 void InitPlugin(JApplication* app) {
   InitJANAPlugin(app);
 
   using namespace eicrecon;
 
-  //-------------------------------------------------------------------------
-  // FarBackward PID Through Phase Space
-  //-------------------------------------------------------------------------
-  PhaseSpacePIDConfig phase_space_pid_cfg{
-      .system        = "TaggerTracker_ID",
-      .direction     = {0.0, 0.0, -1.0},  // Direction is along z-axis
-      .opening_angle = 12 * dd4hep::mrad, // Beampipe opening angle
-      .pdg_value     = 11,                // Set PID to electron
-  };
-
-  app->Add(new JOmniFactoryGeneratorT<PhaseSpacePID_factory>(
-      "FarBackwardTruthSeededPhaseSpacePID",
-      {
-          "ReconstructedTruthSeededChargedWithoutPIDParticles",
-          "ReconstructedTruthSeededChargedWithoutPIDParticleAssociations",
-      },
-      {
-          "ReconstructedTruthSeededChargedWithFBPIDParticles",
-          "ReconstructedTruthSeededChargedWithFBPIDParticleAssociations",
-          "FarBackwardTruthSeededPhaseSpacePIDParticleIDs",
-      },
-      phase_space_pid_cfg, app));
-
-  app->Add(new JOmniFactoryGeneratorT<PhaseSpacePID_factory>(
```

Comment:
Was that the only use of PhaseSpacePID_factory. If yes, let's consider removing it.